packages/components/TopMenu/WalletView.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { StyleProp, View, ViewStyle } from "react-native";
+
+import { Wallet } from "../../context/WalletsProvider";
+import { useNSUserInfo } from "../../hooks/useNSUserInfo";
+import { BrandText } from "../BrandText";
+import { WalletProviderIcon } from "../WalletProviderIcon";
+
+export const tinyAddress = (
```

Comment:
This is a kinda util function and can be reused so we can put this in ex: ```/TopMenu/utils.ts```

Reply:
Done